app/code/core/Mage/Api/Model/Resource/Role.php

```diff
@@ -44,27 +44,21 @@ protected function _construct()
     }
 
     /**
-     * Action before save
-     *
-     * @param Mage_Core_Model_Abstract $object
-     * @return $this
+     * @inheritDoc
      */
-    protected function _beforeSave(Mage_Core_Model_Abstract $object)
+    protected function _beforeSave(Mage_Core_Model_Abstract $role)
     {
-        if (!$object->getId()) {
-            $object->setCreated(now());
+        if ($role instanceof Mage_Admin_Model_Role) {
```

Comment:
That looks like more then just an IDE improvement, did I miss the description in the PR for this?

Reply:
No, wasnt mentioned in this PR. Updated <sup>(1)</sup>.

This only applies to methods that inherit from abstract `Mage_Core_Model_Resource_Db_Abstract` class like `_beforeSave` etc ... 

Renaming wasnt neccessry, just for better readablity ... Is it possible that `$object` is something else then `Mage_Admin_Model_Role` in this case - if this was your question?